problems/4.arrays.js

```diff
@@ -2,8 +2,9 @@
 * @param {number[]} arr - The input array
 * @returns {number} - The average of the first and last values
 */
-function firstAndLastAverage() {
-
+function firstAndLastAverage(arr) {
+  let arr = [23, 14]
```

Comment:
This is hard coded. We want this work for the array that's being passed in. Not for just a random array like [23, 14].